site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -37,70 +40,74 @@ The directory holding the manifest file can be added as an extension in develope
 current state.
 
 1.  Open the Extension Management page by navigating to `chrome://extensions`.
-    - The Extension Management page can also be opened by clicking on the Chrome menu, hovering over
-      **More Tools** then selecting **Extensions**.
+    - Alternatively, open this page by clicking on the Extensions menu button and selecting **Manage
+      Extensions** at the bottom of the menu.
+    - Alternatively, open this page by clicking on the Chrome menu, hovering over **More Tools**
+      then selecting **Extensions**
 2.  Enable Developer Mode by clicking the toggle switch next to **Developer mode**.
-3.  Click the **LOAD UNPACKED** button and select the extension directory.
+3.  Click the **Load unpacked** button and select the extension directory.
 
 {% img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/iYdLKFsJ1KSVGLhbLRvS.png",
        alt="Load Extension", height="337", width="606" %}
 
 Ta-da! The extension has been successfully installed. Because no icons were included in the
-manifest, a generic toolbar icon will be created for the extension.
+manifest, a generic icon will be created for the extension.
 
-## Add instruction {: #background }
+## Add functionality {: #background }
 
-Although the extension has been installed, it has no instruction. Introduce a [background script][9]
-by creating a file titled `background.js`, or downloading it [here][10], and placing it inside the
-extension directory.
+The extension is now installed, but it doesn't currently do anything becasue we haven't told it what
+to do or when to do it. Let's fix that by adding some code to store a background color value.
 
-Background scripts, and many other important components, must be registered in the manifest.
-Registering a background script in the manifest tells the extension which file to reference, and how
-that file should behave.
+To do this, we will need to crate a [background script][1] and add it to the extension's manifest.
+Start by creating a file named `background.js` inside the extension's directory.
 
-```json
+```json/5-7
 {
   "name": "Getting Started Example",
-  "version": "1.0",
   "description": "Build an Extension!",
+  "version": "1.0",
+  "manifest_version": 3,
   "background": {
-    "scripts": ["background.js"],
-    "persistent": false
-  },
-  "manifest_version": 3
+    "service_worker": "background.js"
+  }
 }
 ```
 
-The extension is now aware that it includes a non-persistent background script and will scan the
-registered file for important events it needs to listen for.
+Background scripts, and many other important components, must be registered in the manifest.
```

Comment:
Grammar: either remove commas or have "Background scripts, like many other important components,..."

Reply:
Went with the rewrite suggestion. Resolved in next commit.